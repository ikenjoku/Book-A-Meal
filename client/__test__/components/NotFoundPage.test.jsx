import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFound';


test('it should render 404 Page correctly', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
