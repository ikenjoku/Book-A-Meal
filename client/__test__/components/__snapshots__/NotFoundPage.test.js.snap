// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render 404 Page correctly 1`] = `
<h3>
  <p>
    Nothing here to see
  </p>
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</h3>
`;
